File: app/src/main/java/com/example/bmi/StartingScreen.java
package com.example.bmi;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;

public class StartingScreen extends AppCompatActivity {

    private static int SPLASH_SCREEN_TIME_OUT = 2000; // 2 seconds

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_starting_screen);

        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                Intent menuIntent = new Intent(StartingScreen.this, MenuActivity.class);
                startActivity(menuIntent);
                finish();
            }
        }, SPLASH_SCREEN_TIME_OUT);
    }
}

File: app/src/main/java/com/example/bmi/StartingScreen.kt
package com.example.bmi

import androidx.appcompat.app.AppCompatActivity
import android.content.Intent
import android.os.Bundle
import android.os.Handler

class StartingScreen : AppCompatActivity() {
private val SPLASH_SCREEN_TIME_OUT = 2000 // 2 seconds

        override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_starting_screen)

        Handler().postDelayed({
        val menuIntent = Intent(this@StartingScreen, MenuActivity::class.java)
        startActivity(menuIntent)
        finish()
        }, SPLASH_SCREEN_TIME_OUT.toLong())
        }
        }